### PR TITLE
VOOD-167: Get child variables of child variable in case of lazy objects and update vscode dependecies.

### PR DESCRIPTION
**Pull request by GinoCardillo-OST**

### CODE REVIEW

File: src/debug-adapter/debugEventManager.ts [hunk 1]
```diff
@@ -151,7 +151,10 @@ export class DebugEventManager {
     }
 
     if (isNewAndObject && variable.variablesReference) {
-      const childVariables = await this.debugSessionProxy?.getVariables(variable.variablesReference);
+      let childVariables = await this.debugSessionProxy?.getVariables(variable.variablesReference);
+      if (variable.presentationHint?.lazy && childVariables && childVariables.length > 0) {
```
Comment (alexandre-lagadec-ost) [inline src/debug-adapter/debugEventManager.ts:155]: We could write
```suggestion
      if (variable.presentationHint?.lazy && childVariables?.length > 0) {
```
here, if conciseness is preferred, but 
```typescript
      if (variable.presentationHint?.lazy && childVariables && childVariables.length > 0) {
```
is easy to understand.

File: src/debug-adapter/debugEventManager.ts [hunk 2]
```diff
@@ -151,7 +151,10 @@ export class DebugEventManager {
...
+        childVariables = await this.debugSessionProxy?.getVariables(childVariables[0].variablesReference);
```
Comment (alexandre-lagadec-ost) [inline src/debug-adapter/debugEventManager.ts:156]: Is it sufficient to fetch the child variables from `childVariables[0]` or are there possibly more child variables to be fetched from `childVariables[index]` for indices ranging from `1` to `childVariables.length - 1`?

File: src/debug-adapter/debugEventManager.ts (hunk 1 above)
Comment (GinoCardillo-OST) [reply src/debug-adapter/debugEventManager.ts:155]: I had that idea as well but VS Code doesn't understand that for some reason:
image

File: src/debug-adapter/debugEventManager.ts (hunk 2 above)
Comment (GinoCardillo-OST) [reply src/debug-adapter/debugEventManager.ts:156]: There is only one element.

**Review by alexandre-lagadec-ost — APPROVED** [review]

**Review by PascalSchuermannOst — APPROVED** [review]

**Pull request merged**